syntax = "proto3";

package teos.protobuf.protos.v1;

import "appointment.proto";
import "user.proto";
import "google/protobuf/empty.proto";

service TowerServices {
  rpc register(RegisterRequest) returns (RegisterResponse) {}
  rpc add_appointment(AddAppointmentRequest) returns (AddAppointmentResponse) {}
  rpc get_appointment(GetAppointmentRequest) returns (GetAppointmentResponse) {}
  rpc get_all_appointments(google.protobuf.Empty) returns (GetAllAppointmentsResponse) {}
}